docs/manual/guide/production/ConductR.md

```diff
@@ -26,63 +48,92 @@ The above creates what is known as a "bundle". A bundle is the unit of deploymen
 ...
-[info] Bundle has been created: .../chirper/target/configuration-bundle/cassandra-configuration-06f3e5872f48d69ee339b0a4b7ae382871b69de1cfc1ab831b0a18064d096733.zip
+[info] Bundle has been created: .../my-project/target/configuration-bundle/cassandra-configuration-06f3e5872f48d69ee339b0a4b7ae382871b69de1cfc1ab831b0a18064d096733.zip
 [success] Total time: 4 s, completed 05/03/2016 2:43:07 PM
 ```
 
-You can then load both Cassandra and the configuration for your project into ConductR:
+You can then load the Cassandra bundle with your Cassandra bundle configuration on to ConductR. Note that in this case `cassandra` represents the bundle and the `cassandra-configuration-<hash>.zip` file is the bundle configuration. Copy the configuration zip file from the console output of the previous command.
 
 ```console
-> conduct load cassandra .../chirper/target/configuration-bundle/cassandra-configuration-06f3e5872f48d69ee339b0a4b7ae382871b69de1cfc1ab831b0a18064d096733.zip
+> conduct load cassandra .../my-project/target/configuration-bundle/cassandra-configuration-06f3e5872f48d69ee339b0a4b7ae382871b69de1cfc1ab831b0a18064d096733.zip
 ```
 
-Upon loading you then run cassandra at the scale that you require. For convenience we recommend that you start with one Cassandra cluster per root sbt project, which of course can contain many Lagom projects (and therefore services). Bounded contexts are always maintained via separate key-spaces, and so having one Cassandra cluster is viable for supporting many microservices. The actual number of Cassandra clusters required will be the _Lagom amount_ i.e. "just the right amount" for your system. For more information on configuring Cassandra for ConductR please visit [the bundle's website](https://github.com/typesafehub/conductr-cassandra#conductr-cassandra).
+The tab completion of `conduct load` only works with bundles of the project. The Cassandra bundle is an external bundle hosted on bintray. Therefore tab completion doesn't work in this case.
+
+To run the cassandra bundle execute:
+
+```
+> conduct run cassandra
+```
 
-## For more information
+If the Cassandra bundle has been started on ConductR after the Lagom service itself then it will take a couple of seconds until the Lagom service connects to Cassandra.
 
-For more information on ConductR please visit its [documentation site](https://conductr.lightbend.com/).
+For convenience we recommend that you start with one Cassandra cluster per root sbt project, which of course can contain many Lagom projects (and therefore services). Bounded contexts are always maintained via separate key-spaces, and so having one Cassandra cluster is viable for supporting many microservices. The actual number of Cassandra clusters required will be the _Lagom amount_ i.e. "just the right amount" for your system. For more information on configuring Cassandra for ConductR please visit [the bundle's website](https://github.com/typesafehub/conductr-cassandra#conductr-cassandra).
```

Comment:
> The actual number of Cassandra clusters required will be the _Lagom amount_ i.e. "just the right amount" for your system

That seems a bit like avoiding to answer the question to me. As a user, I think I'd be a bit frustrated of reading such answer without a motivation of what are the variables that one needs to consider.